### ISSUE TITLE
Attempting to create Windows To Go USB results in bricked drive

### ISSUE THREAD

**Issue by jdarn** [pbatard/rufus#2335]
Checklist
---------
- [X] I looked at https://github.com/pbatard/rufus/wiki/FAQ to see if my question has already been answered.
- [X] I performed a search in the issue tracker for similar issues using keywords relevant to my problem, such as the error message I got from the log.
- [X] I clicked the 'Log' button (🗒️) or pressed <kbd>Ctrl</kbd>-<kbd>L</kbd> in Rufus, or used [DebugView](https://learn.microsoft.com/en-us/sysinternals/downloads/debugview), and copy/pasted the log into the section that says `<FULL LOG>` below.
- [X] The log I am copying is the FULL log, starting with the line `Rufus version: x.y.z` - I have NOT removed any part of it.

Additionally (if applicable):
- [X] I ran a bad blocks check, by clicking _Show advanced format options_ then _Check device for bad blocks_, and confirmed that my USB is not defective.
- [X] I also tried one or more of the following:
  - [X] Using a different USB drive.
  - [X] Plugging the USB into a different port.
  - [ ] Running Rufus on a different computer.
- [ ] If using an image, I clicked on the `(✓)` button to compute the MD5, SHA1 and SHA256 checksums, which are therefore present in the log I copied. I confirmed, by performing an internet search, that these values match the ones from the official image.

Issue description
-----------------
Attempting to create a Windows To Go USB has resulted in 2 flash drives being completely bricked. I cannot perform bad block checks, I cannot perform filesystem checks, they are not recognized in the Windows parition manager, they are not recognized as being usable flash media in Linux using GParted, fdisk, or any other such tools. You can see in the logs where the originally dead drive is failing to be read, where I switch to the new flash drive, and where it too fails and from that point forward fails to be recognized as the same device. For reference the Windows 11 install is fresh, it was installed just two days ago. The media being used was just downloaded yesterday from Microsoft and tested to create a normal unmodified Windows installer USB without issue. The flash drives also are new, purchased within the past month, and have already been used to successfully do normal Windows installs, Linux live environments, and Linux installers. It seems like this issue is specific to Windows To Go.

Log
---
Full log was too long for comment, I've attached the file. I'll include what I believe to be the relevant sections in the code blocks below.
```
Rufus x64 v4.2.2074 (Portable)
Windows version: Windows 11 Pro x64 (Build 22621.2361)
Syslinux versions: 4.07/2013-07-25, 6.04/pre1
Grub versions: 0.4.6a, 2.06
System locale ID: 0x0409 (en-US)
Will use default UI locale 0x0409
SetLGP: Successfully set NoDriveTypeAutorun policy to 0x0000009E
Localization set to 'en-US'
Notice: The ISO download feature has been deactivated because 'Check for updates' is disabled in your settings.
Found 517 officially revoked UEFI bootloaders from embedded list
Found 2351 additional revoked UEFI bootloaders from this system's SKUSiPolicy.p7b
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'VendorC ProductCode USB Device' (346D:5678)
Using 'autorun.inf' label for drive F: 'Ubuntu 22.04.2 LTS amd64'
1 device found
Disk type: Removable, Disk size: 64 GB, Sector size: 512 bytes
Cylinders: 7648, Tracks per cylinder: 255, Sectors per track: 63
Partition type: GPT, NB Partitions: 1
Disk GUID: {28A665D3-19D3-4B16-A42F-1E925E3D9D68}
Max parts: 128, Start Offset: 17408, Usable = 62914525696 bytes
Partition 1:
  Type: Microsoft Basic Data Partition
  Name: 'Main Data Partition'
  Detected File System: FAT32
  ID: {DAA79ECE-6158-4DD2-90EB-226CFCCA5A75}
  Size: 58.6 GB (62913478656 bytes)
  Start Sector: 2048, Attributes: 0x0000000000000000
Scanning image...
ISO analysis:
  Image is a UDF image
Disk image analysis:
  Image does not have a Boot Marker
ISO label: 'CCCOMA_X64FRE_EN-US_DV9'
  Detected: Windows 11 ISO (Build 22621.1702)
  Size: 5.4 GB (Projected)
  Has a >4GB file
  Uses: EFI
  Uses: Bootmgr (BIOS and UEFI)
  Uses: Install.wim (version 0.13.1)
Using image: Win11_22H2_English_x64v2.iso (5.4 GB)
WIM extraction method(s) supported: wimgapi.dll
WIM apply method supported: wimgapi.dll
Opening: \\.\CDROM0/sources/install.wim:[0] (API)
Extracting: C:\Users\jackd\AppData\Local\Temp\Ruf69E7.tmp (From [1].xml)
Closing: \\.\CDROM0/sources/install.wim
Will use 'Windows 11 Pro' (Build: 22621, Index 6) for Windows To Go
Selected Windows User Experience options:
• Bypass online account requirement
• Disable data collection
• Use 'jd' for local account name
• Use the same regional options as this user's

Format operation started
Requesting disk access...
Will use 'F:' as volume mountpoint
Opened \\.\PhysicalDrive5 for shared write access
Analyzing existing boot records...
Drive has a Zeroed Master Boot Record
Clearing MBR/PBR/GPT structures...
Erasing 128 sectors
Initializing disk...
Partitioning (GPT)...
● Creating EFI System Partition (offset: 1048576, size: 260 MB)
● Creating Microsoft Reserved Partition (offset: 273678336, size: 128 MB)
● Creating Main Data Partition (offset: 407896064, size: 58.2 GB)
Waiting for logical drive to reappear...
Formatting to NTFS (using IFS)
Using cluster size: 4096 bytes
Quick format was selected
Creating file system...
Format completed.
Opened \\.\PhysicalDrive5 for shared write access
Writing Master Boot Record...
Using Rufus protective MBR
Writing protective message SBR
Found volume \\?\Volume{0eade067-115e-4404-bf7d-35058721f27a}\
Disabling file indexing...
Successfully remounted \\?\Volume{0eade067-115e-4404-bf7d-35058721f27a}\ as F:
Applying Windows image...
Windows To Go mode selected
Mounted ISO as '\\.\CDROM0'
Opening: \\.\CDROM0/sources/install.wim:[6]
Applying Windows image...
Creating: F:\$Recycle.Bin
....
Extracting: F:\Program Files\WindowsApps\microsoft.windowscommunicationsapps_16005.14326.20544.0_x64__8wekyb3d8bbwe\images\contrast-white\GenericMailMediumTile.scale-200.png (603 bytes)
WIM processing error: F:\Program Files\WindowsApps\Microsoft.WindowsAlarms_1.0.75.0_x64__8wekyb3d8bbwe\Assets\AlarmsAppList.targetsize-16_contrast-black.png [[0x00000015] The device is not ready.]
  Could not apply image: [0x00000015] The device is not ready.
Closing: \\.\CDROM0/sources/install.wim
Failed to apply Windows To Go image

Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'VendorC ProductCode USB Device' (346D:5678)
1 device found
No volume information for drive 0x85
Disk type: Removable, Disk size: 64 GB, Sector size: 512 bytes
Cylinders: 7648, Tracks per cylinder: 255, Sectors per track: 63
Partition type: GPT, NB Partitions: 3
Disk GUID: {0F150801-49C1-40D5-8044-C6A6EC0ECC79}
Max parts: 128, Start Offset: 17408, Usable = 62914525696 bytes
Partition 1:
  Type: EFI System Partition
  Name: 'EFI System Partition'
  Detected File System: (Unrecognized)
  ID: {80C28A87-18B4-4756-A888-4034B0D8C53D}
  Size: 260 MB (272629760 bytes)
  Start Sector: 2048, Attributes: 0x0000000000000000
Partition 2:
  Type: Microsoft System Reserved Partition
  Name: 'Microsoft Reserved Partition'
  Detected File System: (Unrecognized)
  ID: {C7E7EFFE-7B8D-4601-8D03-C333FF812703}
  Size: 128 MB (134217728 bytes)
  Start Sector: 534528, Attributes: 0x0000000000000000
Partition 3:
  Type: Microsoft Basic Data Partition
  Name: 'Main Data Partition'
  Detected File System: NTFS
  ID: {0EADE067-115E-4404-BF7D-35058721F27A}
  Size: 58.2 GB (62506643456 bytes)
  Start Sector: 796672, Attributes: 0x0000000000000000
WIM extraction method(s) supported: wimgapi.dll
WIM apply method supported: wimgapi.dll
Opening: \\.\CDROM0/sources/install.wim:[0] (API)
Extracting: C:\Users\jackd\AppData\Local\Temp\RufE577.tmp (From [1].xml)
Closing: \\.\CDROM0/sources/install.wim
Will use 'Windows 11 Pro' (Build: 22621, Index 6) for Windows To Go
Selected Windows User Experience options:
• Bypass online account requirement
• Disable data collection
• Use 'jd' for local account name
• Use the same regional options as this user's

Format operation started
Requesting disk access...
Will use 'F:' as volume mountpoint
Opened \\.\PhysicalDrive5 for shared write access
Analyzing existing boot records...
Drive has a Zeroed Master Boot Record
Clearing MBR/PBR/GPT structures...
Erasing 128 sectors
Initializing disk...
Partitioning (GPT)...
● Creating EFI System Partition (offset: 1048576, size: 260 MB)
● Creating Microsoft Reserved Partition (offset: 273678336, size: 128 MB)
● Creating Main Data Partition (offset: 407896064, size: 58.2 GB)
Waiting for logical drive to reappear...
Formatting to NTFS (using IFS)
Using cluster size: 4096 bytes
Quick format was selected
Creating file system...
Format completed.
Opened \\.\PhysicalDrive5 for shared write access
Writing Master Boot Record...
Using Rufus protective MBR
Writing protective message SBR
Found volume \\?\Volume{722111f6-b6a3-40c2-8fec-d73dc38e5d88}\
Disabling file indexing...
Successfully remounted \\?\Volume{722111f6-b6a3-40c2-8fec-d73dc38e5d88}\ as F:
Applying Windows image...
Windows To Go mode selected
Mounted ISO as '\\.\CDROM0'
Opening: \\.\CDROM0/sources/install.wim:[6]
Applying Windows image...
Creating: F:\$Recycle.Bin
...
Extracting: F:\Program Files\WindowsApps\Microsoft.WindowsAlarms_1.0.75.0_x64__8wekyb3d8bbwe\Assets\AlarmsAppList.targetsize-64_contrast-white.png (823 bytes)
WIM processing error: F:\Program Files\WindowsApps\Microsoft.WindowsAlarms_1.0.75.0_x64__8wekyb3d8bbwe\Assets\AlarmsAppList.targetsize-64_contrast-white.png [[0x000003EE] The volume for a file has been externally altered so that the opened file is no longer valid.]
WIM processing error: F:\Program Files\WindowsApps\Microsoft.People_10.1909.12456.0_neutral_split.scale-125_8wekyb3d8bbwe\Assets\contrast-white\PeopleWideTile.scale-125.png [[0x00000015] The device is not ready.]
WIM processing retry: F:\Program Files\WindowsApps\Microsoft.PowerAutomateDesktop_1.0.219.0_neutral_split.scale-180_8wekyb3d8bbwe\AppxManifest.xml [[0x00000015] The device is not ready.]
WIM processing error: F:\Program Files\WindowsApps\Microsoft.PowerAutomateDesktop_1.0.219.0_neutral_split.scale-180_8wekyb3d8bbwe\AppxManifest.xml [[0x00000015] The device is not ready.]
  Could not apply image: [0x000003EE] The volume for a file has been externally altered so that the opened file is no longer valid.
Closing: \\.\CDROM0/sources/install.wim
Failed to apply Windows To Go image

Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
Found USB 3.0 device 'FC 3281B USB Device' (346D:3281)
Device eliminated because it appears to contain no media
0 devices found
Found UAS (USB 3.1) device 'WD Game Drive UAS Device' (1058:2653)
Device eliminated because it was detected as a Hard Drive (score 43 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
0 devices found
Saved C:\Users\jackd\rufus.log
```
[rufus.log](https://github.com/pbatard/rufus/files/12838187/rufus.log)

**Comment by jdarn** [comment]
Verified SHA256 hash of the iso manually just now in Windows Terminal: 
`8059A99B8902906A90AFE068AC00465C52588C2BD54F5D9D96C1297F88EF1076`

This matches the hash on the table provided by Microsoft on their downloads page.

**Comment by pbatard** [comment]
1. If you check the issue tracker or other places, you will see that you are the **only** person to report this kind of issue.
2. Rufus is downloaded about 3 million times each month, and a not so small fraction of these downloads are being used to create Windows To Go drives
3. A lot of those will have been created using the latest English version of the Windows 11 22H2 ISO that has been released since 2023.05.

__Therefore__, we can safely conclude that the issue is not with the application, because if it was there would be tons of report of "bricked" drives from the thousands of people who have been using Rufus to create a Windows To Go drive with the latest Windows 11 22H2 English ISO release.

Which means that this is an issue with your environment or your hardware.

The fact that it your drives appeared to work okay in some specific scenarios and not with Windows To Go could be due to a number of factors, such as poorly designed drives (since the Windows To Go image application process does a lot of rapid random I/O which may stress a drive a lot more than the usual sequential I/O), the use of "fake" drives (such as an 8 GB masquerading as a 64 GB drive), flaky USB connectivity and so on...

As a matter of fact, since all Rufus really does for the creation of Windows To Go drive, is pretty much follow [these official steps exactly](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-8.1-and-8/jj721578(v=ws.11)), I am confident that your drives would have failed regardless of whether you used Rufus or not because, once again, the issue is coming from your environment.

Besides, there is absolutely nothing we have control on during the image application phase, where you experience the failure, as Rufus is simply telling the Windows internal API to apply the `sources\install.wim` image to the target drive, and this whole process, __which is controlled exclusively by Microsoft__ is a black box (i.e. Rufus is not copying the files here, it's merely asking Windows to do so and report on the progress).

Finally, you probably want to read [this](https://github.com/pbatard/rufus/wiki/FAQ#user-content-Help_Rufus_damaged_my_flash_drive).

Now, I appreciate that this is most likely not the answer that you want to read, and that you will want to consider that two drives failing during the same process must be indicative of a problem with the application that initiated that process, but at the very least, the lack of user reports similar to yours, of which there should be plenty if there __really__ was an issue with Rufus, should give you some pause as to whether Rufus is actually to blame...

**Comment by jdarn** [comment]
Don't worry about it, I'm not too bothered about the whole thing given they were pretty cheap disposable flash drives. I just wanted to make sure that it wasn't a mistake on my part. Given the type of failure observed here, I'm willing to accept that whatever blackbox process MS uses for Windows To Go was just too much for either the controllers or the NAND flash on these flash drives, since the controllers have effectively gone into dumb mode. 

Thanks for taking the time to reply on the weekend.
